### DISCUSSION TITLE
Question regarding re-submitting solution

### DISCUSSION THREAD

**Discussion by adrianymeri** [esa/SpOC3#12] [Challenge 1: Torso Decompositions] [answered]
Hi SpOC3,
Just submitted a solution for small-graph, but would love to have the chance of resubmitting it again, together with medium-graph and large-graph solution.

**Comment by maxbannach** [comment, marked as answer]
Hi!
I may not fully understand the question. The solutions are not limited, so you can submit them again. You can also independently submit solutions to the various graph sizes.
